ceph/ceph_admin/helper.py

```diff
@@ -763,6 +764,34 @@ def file_or_path_exists(node, file_or_path):
     return False
 
 
+def file_or_path_updated(node, file_or_path, interval=120):
+    """
+    Method to check if the given file or directory was updated in the given interval
+    Args:
+        node: node object where file should be exists
+        file_or_path: ceph file or directory path
+        interval: if mentioned checks whether the file existing in the given path
+                              was updated anytime between now and the interval given here.
+
+    Returns:
+        boolean
+    """
+    try:
+        out, _ = node.exec_command(cmd=f"date -u -r {file_or_path}", sudo=True)
+        LOG.info(f"Output : {out}")
+    except CommandFailed as err:
+        LOG.error(f"Error: {err}")
+        return False
+
+    modified_time = parser.parse(out).replace(tzinfo=None)
+    time_now = datetime.utcnow().replace(tzinfo=None)
+    time_diff = (time_now - modified_time).seconds
+    LOG.info(f"Time difference between now and gz file modified date : {time_diff}")
+    if time_diff > interval:
+        return False
```

Comment:
Better to add log.error here with expected and actual value.

Reply:
Good catch. thanks Vaibhav. Added the error log.